Spanner/src/Connection/Grpc.php

```diff
@@ -166,10 +183,13 @@ public function __construct(array $config = [])
             $grpcConfig['apiEndpoint'] = $config['apiEndpoint'];
         }
 
-        $this->spannerClient = isset($config['gapicSpannerClient'])
-            ? $config['gapicSpannerClient']
-            : $this->constructGapic(SpannerClient::class, $grpcConfig);
+        $this->enableResourceCaching = isset($config['enableCaching']) && $config['enableCaching'];
+        $this->endpointUris=array();
+        $this->spannerClients=array();
```

Comment:
No preference here, however I think in this library
```php
= [];
```
array initialization is preferred

Reply:
I agree. Replaced.